src/sentry/static/sentry/app/components/alertMessage.jsx

```diff
@@ -31,7 +32,7 @@ const AlertMessage = React.createClass({
             <span aria-hidden="true">&times;</span>
           </button>
           <span className="icon"></span>
-          {this.props.message}
+          {this.props.url ? <a href={this.props.url}>{this.props.message}</a> : this.props.message}
```

Comment:
This isn't a big deal tbh, but I prefer doing newlines often for these kinds of things.

```
{this.props.url ?
   <a href={this.props.url}>{this.props.message}</a> 
:
   this.props.message
}
```